### DISCUSSION TITLE
[Error] the --url option is mandatory

### DISCUSSION THREAD

**Discussion by swethabikkasani** [jfrog/jfrog-cli#2009] [Q&A]
Hi,

I am getting the following error while trying to run a command:

Commands: 
jf c add $SERVER_ID --url=$JF_URL --access-token=$JF_ACCESS_TOKEN --interactive=false
jf rt upload --flat=true '/path/of/file/to/upload' jfrog-repo/ 

[Error] the --url option is mandatory

Can you please let me know what is the issue here and the resolution.

Thanks,
Swetha

**Comment by sverdlov93** [comment]
Hi @swethabikkasani ,
Can you run `jf c show` and see if you have another server configs?
My guess is that you have another config that has no URL and is defined as the default
You can run `jf c use $SERVER_ID` to make your config default or add the server-id flag to the upload command: `jf rt upload --flat=true '/path/of/file/to/upload' jfrog-repo/ --server-id=$SERVER_ID`
